packages/integration-tests/src/test-project-setup/cdk/create_empty_cdk_project.ts

```diff
@@ -24,5 +25,10 @@ export const createEmptyCdkProject = async (
 
   await cdkCli(['init', 'app', '--language', 'typescript'], projectRoot).run();
 
+  await execa('npm', ['install', '-D', '@aws-amplify/auth-construct-alpha'], {
```

Comment:
this doesn't seem like it belongs in "create_empty_cdk_project". Maybe it should be part of the `createProject` method in `AuthTestCdkProjectCreator`?

Reply:
I think we can skip "raw cdk tests" in project manager expansion and just test them with npm.
The reason is that these tests are only focused on our L3 constructs and we'd be just testing CDK CLI here.